torch/ao/sparsity/experimental/data_sparsifier/base_data_sparsifier.py

```diff
@@ -0,0 +1,76 @@
+from typing import Optional, Tuple, List, Any, Dict
+from ...sparsifier import base_sparsifier
+from collections import defaultdict
+from torch import nn
+
+
+class Container(nn.Module):
+    def __init__(self):
+        super().__init__()
+
+
+class BaseDataSparsifier(base_sparsifier.BaseSparsifier):
+    r"""
+    Base Data Sparsifier class for all Data sparsifiers.
+    The abstract class accepts raw torch tensors / embedding / embedding bags (refer to SUPPORTED_TYPES above)
+    to prepare for sparsification.
+    In this case, mask (and parametrizations) is owned by the class and not by the user.
+    Specifically, the container object inside the class maintains the mask and parametrizations of the input data
+    Args:
```

Comment:
Space before `Args:`. See [documentation here](https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html?highlight=Args#type-annotations) (refer to the "google style")

Reply:
Noted

Reply:
example: https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/conv.py#L671